### PR TITLE
add basscss w/ npm autoupdate

### LINKED ISSUE #8239: [Request] Add BassCSS
**Library name:**  BassCSS
**Git repository url:** https://github.com/basscss/basscss/
**npm package url(optional):** https://www.npmjs.com/package/basscss
**License(s):** MIT
**Official homepage:** http://www.basscss.com/

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #8239 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/basscss/basscss/
 * Official website (optional, not the repository):  http://basscss.com
 * NPM package url (optional): https://www.npmjs.com/package/basscss
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 3313
   - Count of watchers: 117
   - Count of forks: 216
 * NPM download stats (optional):
   - Downloads in the last day: 73
   - Downloads in the last week: 2258
   - Downloads in the last month: 9590

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8239 , cc @jxnblk

### CODE REVIEW

File: ajax/libs/basscss/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "css",
+    "framework",
+    "oocss",
+    "basscss"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/basscss/basscss.git"
+  },
+  "homepage": "https://www.basscss.com/",
```
Comment (x09326) [inline ajax/libs/basscss/package.json]: I think they don't provide ```https```.

**Comment by pvnr0082t** [comment]
@x09326 I have modified homepage field. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t have you confirmed with the author about the possibility of providing minified basscss-important.css officially?

**Comment by pvnr0082t** [comment]
No. I'll confirm that.

**Comment by pvnr0082t** [comment]
@x09326 author provide minified basscss-important.css in v8.0.2. Please help me review again, thank you.

**Comment by PeterDaveHello** [comment]
@pvnr0082t the version in branch name is wrong, and I'll suggest you just remove it, since version may changed during the review, and you added other versions.

**Pull request merged**